README.rst

```diff
@@ -20,41 +29,70 @@ simtools
 .. image:: https://codecov.io/gh/gammasim/simtools/branch/master/graph/badge.svg?token=AYAIRPARCH
    :target: https://codecov.io/gh/gammasim/simtools
 
+Simulation tools and applications for the `Cherenkov Telescope Array (CTAO) <www.cta-observatory.org>`_.
+
+License: `BSD-3 <https://github.com/gammasim/simtools/blob/main/LICENSE>`_
+
+Useful links:
+`Documentation <https://gammasim.github.io/simtools/>`_
+`Source code <https://github.com/gammasim/simtools>`_
+`Issue tracker <https://github.com/gammasim/simtools/issues>`_
+`Citation <https://github.com/gammasim/simtools/blob/main/CITATION.cff>`_
 
-Prototype implementation of tools for the Simulation Pipeline of the `CTA Observatory <www.cta-observatory.org>`_:
+simtools provides:
 
-* libraries for simulation model management and model database interface;
-* applications for simulation model parameter derivation and validation;
-* standardized interfaces and data products independent of the underlying simulation software (e.g., `CORSIKA <https://www.iap.kit.edu/corsika/>`_, `sim_telarray <https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/>`_);
-* I/O and reporting tools for the MC simulation model parameter and production database.
+* libraries for simulation model management and model database interface
+* tools for the preparation and configuration of simulation productions
+* applications for simulation model parameter derivation and validation
+* standardized interfaces and data products independent of the underlying simulation software (e.g., `CORSIKA <https://www.iap.kit.edu/corsika/>`_, `sim_telarray <https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/>`_)
+* I/O and reporting tools for the MC simulation model parameter and production database
 
 simtools is one part of the CTAO Simulation Pipeline, which consist of the following components:
 
-- `CORSIKA <https://www.iap.kit.edu/corsika/>`_ air shower simulation code and the `sim_telarray <https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/>`_ telescope simulation code.
-- simtools for tools and application for model parameter and configuration management, database interfaces, validation, and production preparations.
-- databases, especially the model parameter database.
+- `CORSIKA <https://www.iap.kit.edu/corsika/>`_ air shower simulation code and the `sim_telarray <https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/>`_ telescope simulation code
+- `workflows <https://github.com/gammasim/workflows>`_ for setting, derivation and validation of simulation model parameters
+- `simulation model parameter and input data schema <https://github.com/gammasim/workflows>`_
+- databases, especially the model parameter database
```

Comment:
databases is the only one here without a link. Maybe we can link this to `simtools/docs/build/html/databases.html#model-parameters-db`.

Reply:
OK, done!